src/server/controllers/users.controller.js

```diff
@@ -118,6 +118,7 @@ function storeSession(req, res, user) {
   req.session.access_token = req.body.access_token;
   req.session.email = user.email;
   req.session._id = user._id.toString();
+  req.session.date = user.stamp_login;
```

Comment:
There is no code for stamp time at login. You should update DB when login using save().exec() or update().exec().